src/server.tsx

```diff
@@ -104,4 +102,22 @@ const startServer = async () => {
   });
 };
 
+function getOpenRobotsTxt() {
+  const robotstxt = ['User-agent: *'];
+  const AI = GameMode.AI;
+  const OnlineFriend = GameMode.OnlineFriend;
+  GAMES_LIST.forEach((game) => {
+    game.modes.forEach((modeInfo) => {
+      if (modeInfo.mode === GameMode.AI) {
+        robotstxt.push(`Disallow: /g/${game.code}/ai/`);
+      }
+      if (modeInfo.mode === GameMode.OnlineFriend) {
+        robotstxt.push(`Disallow: /g/${game.code}/online/`);
```

Comment:
arent we supposed to have a * in the end?

Reply:
Apparently some robots.txt parsers do not support wildcard syntax.  /online/ and such should work and disallow all subdirectories.